test-cloud/test-cloud-remote/src/test/resources/test.properties

```diff
@@ -14,15 +14,9 @@ sso.service.username=serviceUser
 sso.service.password=serviceUser1!
 
 kie.app.name=myapp
-
-org.kie.server.user=yoda
-org.kie.server.pwd=usetheforce123@
-org.kie.workbench.user=adminUser
-org.kie.workbench.pwd=adminUser1!
-org.kie.server.controller.user=controllerUser
-org.kie.server.controller.pwd=controllerUser1!
-org.kie.workbench.maven.user=mavenUser
-org.kie.workbench.maven.pwd=mavenUser1!
+kie.app.user=adminUser
```

Comment:
also there should be change appUser

Reply:
I missed one commit, thanks for spotting. Fixed.